pkg/volume/host_path/nsenter.go

```diff
@@ -19,53 +19,30 @@ limitations under the License.
 package host_path
 
 import (
-	"fmt"
-
-	"k8s.io/utils/exec"
-)
-
-const (
-	hostProcMountsNamespace = "/rootfs/proc/1/ns/mnt"
-	nsenterCmd              = "nsenter"
-	statCmd                 = "stat"
-	touchCmd                = "touch"
-	mkdirCmd                = "mkdir"
+	"k8s.io/api/core/v1"
```

Comment:
I thought that whole goal of this refactoring is to remove `nsenter.go` from `HostPath` plugin. It should have no platform-dependent code (no host_path_linux.go) and it should use just the mounter provided by kubelet / `VolumeHost`.

What's missing in the mounter? Feel free to add `MakeDir` and `MakeFile` there.